inc/changelog.php

```diff
@@ -83,31 +79,7 @@ function addLogEntry(
     ]);
 
     // update metadata
-    if (!$wasRemoved) {
-        $oldmeta = p_read_metadata($id)['persistent'];
-        $meta    = array();
-        if ($wasCreated &&
-            (empty($oldmeta['date']['created']) || $oldmeta['date']['created'] === $created)
-        ) {
-            // newly created
-            $meta['date']['created'] = $created;
-            if ($user) {
-                $meta['creator'] = isset($INFO) ? $INFO['userinfo']['name'] : null;
-                $meta['user']    = $user;
-            }
-        } elseif (($wasCreated || $wasReverted) && !empty($oldmeta['date']['created'])) {
-            // re-created / restored
-            $meta['date']['created']  = $oldmeta['date']['created'];
-            $meta['date']['modified'] = $created; // use the files ctime here
-            $meta['creator'] = isset($oldmeta['creator']) ? $oldmeta['creator'] : null;
-            if ($user) $meta['contributor'][$user] = isset($INFO) ? $INFO['userinfo']['name'] : null;
-        } elseif (!$minor) {   // non-minor modification
-            $meta['date']['modified'] = $date;
-            if ($user) $meta['contributor'][$user] = isset($INFO) ? $INFO['userinfo']['name'] : null;
-        }
-        $meta['last_change'] = $logEntry;
-        p_set_metadata($id, $meta);
-    }
+    $pageFile->updateMetadata($logEntry);
```

Comment:
There is now twice a call of `updateMetadata()`..

Reply:
The **PageFile::changelog->addLogEntry()** updates `.change` file only in meta folder, but not `.meta` file.

Reply:
I see where I was confused. The two places where updateMetadata() are called is the new location and the deprecated location in the code were this function was used. It is now clear.